src/components/SearchPage.js

```diff
@@ -97,8 +97,9 @@ class SearchPage extends React.Component {
         let sequenceTime = new Date();
 
         // for testing, change cmlgbackend.wdcc to cmlgdevbackend.wdcc
-        let url = 'https://cmlgbackend.wdcc.co.nz/api/translations?sequence=' + sequenceTime.getTime() +
-                  '&pageRows=all';
+        let url = 'https://cmlgdevbackend.wdcc.co.nz/api/translations?sequence=' + sequenceTime.getTime() +
```

Comment:
Can't use dev mode when u pushing to master. Need to change back.

Reply:
Sorry kevin you still need to change this back

Reply:
Please change back